web/wp-content/themes/dctx-v2/assets/bower_components/slick-carousel/slick/slick.js

```diff
@@ -0,0 +1,3011 @@
+/*
```

Comment:
:no_entry_sign: **Error**: You must use &quot;/**&quot; style comments for a file comment (*Squiz.Commenting.FileComment.WrongStyle*).